doc/api/v8.md

```diff
@@ -323,6 +323,10 @@ added: v8.0.0
 
 Uses a [`DefaultSerializer`][] to serialize `value` into a buffer.
 
+[`ERR_BUFFER_TOO_LARGE`][`ERR_BUFFER_TOO_LARGE`] will be thrown when trying to
```

Comment:
I _think_ we generally prefer this, which I assume is equivalent:

```suggestion
[`ERR_BUFFER_TOO_LARGE`][] will be thrown when trying to
```

Reply:
Thank you @tniessen, updated.